README.md

```diff
@@ -87,11 +87,7 @@ release.
 
 The latest container image can be found at:
 * `quay.io/coreos/kube-state-metrics:v1.9.7`
-* `k8s.gcr.io/kube-state-metrics:v1.9.7`
-
-**Note**:
-The recommended docker registry for kube-state-metrics is `quay.io`. kube-state-metrics on
-`gcr.io` is only maintained on best effort as it requires external help from Google employees.
+* `gcr.io/k8s-staging-kube-state-metrics/kube-state-metrics:v1.9.7`
```

Comment:
I would not recommend using staging repo as official, images there can still be manually pushed. 

We should official promotion process which is done through a sending a PR to kubernetes/k8s.io, example: https://github.com/kubernetes/k8s.io/pull/761

Resulting image would be available in k8s.gcr.io/kube-state-metrics/kube-state-metrics:v1.9.7

Reply:
Ah thanks for that piece of info!